ci/setup_ci_environment.sh

```diff
@@ -7,6 +7,9 @@ set -e
 apt-get update
 apt-get install --no-install-recommends --no-install-suggests -y \
                 build-essential \
+                cmake \
+                g++-10 \
+                gcc-10 \
```

Comment:
setup_ci_environment is also used by bazel builds and clang builds.

Please define helpers like:
- setup_cmake.sh
- setup_gcc.sh (or setup_gcc10.sh)

and use them explicitly for platforms that do not provide cmake and gcc by default in the runner, like here.

It will help decouple things, allowing an easier upgrade/downgrade of gcc versions in say ubuntu independendly of arm.

Reply:
thanks, done.